website_sale_product_description/models/product_template.py

```diff
@@ -7,7 +7,6 @@
 _logger = logging.getLogger(__name__)
```

Comment:
remove both _logger variable and import logging. this two are not necessary here